.github/workflows/update-pr.yml

```diff
@@ -0,0 +1,15 @@
+name: "Update Pull Request"
+
+on: pull_request
+
+jobs:
+  update_pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: tzkhan/pr-update-action@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        head-branch-regex: '.*'
+        title-template: ''
+        body-template: '[Preview Tests](https://raw.githack.com/w3c/aria-at/%branch%/index.html)'
```

Comment:
I think you mean `%headbranch%`? The link doesn't work with `%branch%`

Reply:
Opps. This weekend my PR got merged into the action and they did a v2 release which changed the entire action and I just swtiched to it. Good catch. Fixed now.